src/style.css

```diff
@@ -0,0 +1,52 @@
+html {
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+}
+
+body {
+  margin: 0;
+}
+
+.header {
+  height: 60px;
+  padding: 0 50px;
+}
+
+.header,
+.nav {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.nav__link {
+  position: relative;
+  color: #000;
+  text-decoration: none;
+  text-transform: uppercase;
+}
+
+.nav__list {
+  display: flex;
+  list-style: none;
+}
+
+.nav__li {
+  margin: 10px;
+}
```

Comment:
The first link should not have margin-left, and the last link should not have margin-right